### ISSUE TITLE
Refactor Fileset Head Tag Title

### ISSUE THREAD

**Issue by mtribone** [samvera/hyrax#1283]
### Descriptive summary

In Hyrax 2.0.0 alpha, the `<title>` element in the `<head>` of the File Set show page displays "Show File Set // Hyrax." Display `<title>` as "File Set | File Set Title | File Set ID: # | Hyrax" or some form instead.

### Rationale

The SEO for every File Set will be quite poor with every File Set having the same value. I[t also aids with accessibility.](http://accessibility.psu.edu/titleshtml/)

### Expected behavior

I expected the `<title>` in the `<head>` to reflect the File Set Title and its relationship in the system.

### Actual behavior

I get "Show File Set // Hyrax" for every File Set.

### Steps to reproduce the behavior

1. View a File Set
1. Inspect the code
1. Look for the `<title` element in the `<head>`

### Related work

https://github.com/psu-stewardship/scholarsphere/issues/516

**Comment by mtribone** [comment]
We should do this for the Work Show, too. The Work Show just outputs the title of the work, which is good, but would be better for SEO and discovery it included the work ID and application name.

**Comment by h-parekh** [comment]
@mtribone We can close this since its working as expected. I created a [new work in Nurax](https://nurax.curationexperts.com/concern/generic_works/nv9352930?locale=en#?c=0&m=0&s=0&cv=2&xywh=-607%2C0%2C2236%2C1024) and I see this in the html code:
```html
<title>Generic Work | Work with fileset | ID: nv9352930 | Nurax</title>
```

**Comment by h-parekh** [comment]
Confirmed, working as expected.

**Comment by tap87** [comment]
I am reopening this issue should be opened again. This is a problem not on the work show page, but the file set show page. I am seeing the problem on my app as well as nurax.

**Comment by mtribone** [comment]
@vantuyls Yes this is still an issue with the FileSet.

https://nurax-dev.curationexperts.com/concern/parent/zs25x8606/file_sets/r207tp50b

<img width="1433" alt="screen shot 2018-08-09 at 10 26 13 am" src="https://user-images.githubusercontent.com/4163828/43905496-33ae7b4e-9bbf-11e8-84ee-2fc58c181b9d.png">
